go.sum

```diff
@@ -1,8 +1,28 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.31.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
go.mod is not changed, we should leave go.sum also without any changes.

Reply:
i took the so.sum from master.. i dont know how this creaps in.. let me do it again.